index.d.ts

```diff
@@ -125,6 +127,18 @@ export interface Locale {
 
 export type LocaleCollection = ContentfulCollection<Locale>;
 
+export interface Tag {
+    name: string
+    sys: {
```

Comment:
Should we define the full received `sys` object here?
Reference: https://www.contentful.com/developers/docs/references/content-delivery-api/#/reference/content-tags/tag/get-a-single-tag/console/curl

Reply:
It's not done on the other entities, so I'd rather spare the effort of putting that information in. I heard someone was working on some TS refactoring ;)